tasks.py

```diff
@@ -15,263 +15,284 @@
 limitations under the License.
 """
 import os
-from invoke import task  # type: ignore
```

Comment:
Does mypy no longer require the `type: ignore` comment to avoid complaining about invoke?